rc-files/HOW_TO_BUILD.md

```diff
@@ -0,0 +1,5 @@
+Step 1. Run `download_resources.sh` to add the necesary asset files
+
+Step 2. Apply all patches in the `patches` folder
```

Comment:
Could be nice to include the sample command for the one patch we do have (would be this right? `git apply rc-bl_info.patch`)

Reply:
Yeah, although we actually have 2 patches (the other changes the tracker ID so we can better track RCs in the tracker)